### ISSUE TITLE
[Accessibility] Search trap using TAB key

### ISSUE THREAD

**Issue by dingrassia** [adobe/aem-core-wcm-components#1553]
## Bug Report

**Current Behavior**
When you land on a page with the Quick Search component and try to use the TAB key to navigate through the site it will get stuck on the Search input field. This wrong behavior disappears when you click outside the Quick Search input: trying again to navigate the site using the TAB key, it will focus on the input field only once and then it proceeds to the next element as normally expected.

**Expected behavior/code**
Using the TAB key the user should be able to navigate the website and when he/she lands on the Quick Search should proceed to the next element clicking the TAB key another time.

**Environment**
- AEM as a Cloud Service SDK v2021.2.4944.20210221T230729Z-210128
- Core Components version 2.16.0
- Java(TM) SE Runtime Environment 18.9 (build 11.0.6+8-LTS)
The error can be noticed on https://wknd.site too.

**Comment by msagolj** [comment]
Was able to reproduce the behavior with 2.15.2 but not with 2.16.0 , I guess this got fixed with #1437, but please note that:
- 2.16.0 hasn't been rolled out since we found a last-minute bug and are currently fixing and preparing a 2.16.2 patch release
- so wknd.site is most probably running on 2.15.2
- SDK 4944 is running with 2.14
- if you update your SDK QS manually to 2.16.0 make sure you delete the precompiled scripts bundle in/system/console and that only one version of the core component bundles is active .
- 
Closing issue for now. Reopen it if you the issue is still reproducible for you after making sure youre on 2.16.
